src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/DetailControl.xaml

```diff
@@ -116,8 +116,8 @@
             <Hyperlink
               NavigateUri="{Binding Path=PackageMetadata.PackageDetailsUrl}"
               Style="{StaticResource HyperlinkStyle}"
-              AutomationProperties.Name="{x:Static nuget:Resources.Accessibility_PackageDetailsUrlLink}"
```

Comment:
Can we remove the `Accessibility_PackageDetailsUrlLink` resource now?

Reply:
yep. next iteration. thx.